src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,16 +2,22 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] arrayOfRymers = new int[12];
-    public int currentSize = -1;
+    private final int CAPACITY = 12;
+    private final int EMPTY_VALUE = -1;
+    private int[] arrayOfRhymers = new int[CAPACITY];
+    private int currentSize = EMPTY_VALUE;
```

Comment:
name is not precise (size-1)


Extracting this constant is done incorrectly. Should show during next classes ;)